### PR TITLE
Updates for Zammad 5

### PR DESCRIPTION
**Pull request by mgruner**
- Changed Ruby version dependency to 2.7.3
- Add Node.js dependency

### CODE REVIEW

**Comment by mgruner** [comment]
@MrGeneration added info about Redis in 5975b22, does that seem ok?

**Comment by mgruner** [comment]
@MrGeneration added information about Memcached.

**Comment by mantas** [comment]
Ruby version changed once again, to 2.7.4

**Comment by MrGeneration** [comment]
Sorry for the noise I needed the current state in this branch. :D

**Comment by MrGeneration** [comment]
Hey MG,

could you please double tab on my changes? 🙏 

Thanks!

File: appendix/configure-env-vars.rst
Comment (mgruner) [inline appendix/configure-env-vars.rst:135]: The options (listed) below?

File: install/package.rst
Comment (mgruner) [inline install/package.rst:40]: its assets.

File: install/package.rst
Comment (mgruner) [inline install/package.rst:43]: to install Node.js in case (reference seems unclear as package based installs mentioned in the previous sentence)

File: install/package.rst
Comment (mgruner) [inline install/package.rst:184]: an issue

File: prerequisites/software.rst
Comment (mgruner) [inline prerequisites/software.rst:8]: Most of the software versions (listed) below

File: prerequisites/software.rst
Comment (mgruner) [inline prerequisites/software.rst:49]: 👍

File: prerequisites/software.rst
Comment (mgruner) [inline prerequisites/software.rst:207]: The features / integrations (mentioned) below were...

**Review by mgruner — COMMENTED** [review]
@MrGeneration some small grammar/spelling suggestions from my side, the rest looks fine!

Note: I can't approve this PR since I'm the author. I think you can probably approve it. 🚦

File: install/package.rst
Comment (MrGeneration) [reply install/package.rst:43]: Good point, I exchanged ``this package`` with ``node.js``

**Review by MrGeneration — APPROVED** [review]

**Comment by MrGeneration** [comment]
If I happen to oversleep release, feel free to press the merge button at any time needed. 💪

**Pull request merged**
